docs/control_center_docs/control_center_start_guide.rst

```diff
@@ -0,0 +1,737 @@
+******************************************
+The Control Center provides a basic suite of testing options to ensure that your module is functioning properly. These options are available in the testing tab, 
+available on the left side.
+
+.. note::
+    Before using the Control Center's testing functionality, we recommend that you increase your module's timeout parameter to 1.5s. 
+    This parameter is available through the tuning tab. If you do not increase the timeout value, your module may fail to spin as expected.
+
+.. warning::
+    Please remove all propellers before interacting with any testing parameters. Failure to do so is dangerous.
+
+In order to set a testing parameter, simply enter a value into the spin box, and hit the down arrow set button. Parameters such as Coast do not have a spin box, 
+and only require that you press the set button to trigger the behavior.
+
+The testing tab for a Veritq 23-06 2200Kv looks as follows:
+
+.. image:: ../_static/control_center_pics/23_testing_tab.png
+    :scale: 50%
+
+Brake
+=========
+This commands the module to brake, which shorts the motor phases, which slows the motor down by dissipating energy in the motor.
+
+.. warning::
+    If you are using a power supply to power your module, it is possible to damage or destroy your module with aggressive commands, e.g. quickly switching from spinning at full speed to stopping. 
+    This is because Vertiq modules can also act as generators. In general, power supplies, unlike batteries, cannot absorb the energy generated by the module. As such, aggressive commands can lead to dangerous 
+    voltage spikes when connected to a power supply. To prevent damage to the module when commanding it aggressively on a power supply, it is recommended to turn on the regeneration voltage protection feature.
+
+Coast
+=============
+Commanding a coast means that the module allows its rotor to slowly decelerate and come to a stop. It is a safe way to stop the module when testing since it does not cause the module to rapidly decelerate.
+
+PWM
+========
+This tells the module to spin with a percentage of its input voltage. For example, if your input voltage is 20V, and you set this parameter to 0.5, the module will spin with a 10V drive voltage, since 50% of 20V is 10V. A positive PWM percentage always goes in the counterclockwise direction, and a negative PWM percentage always goes clockwise.
+
+Velocity
+=============
+This commands the module to try and maintain a specific velocity in rad/s, using closed loop PID control. A positive velocity always goes in the counterclockwise 
+direction, and a negative velocity always goes clockwise. To learn more about velocity and voltage control 
+see :ref:`manual_velocity_control_mechanisms`
+
+Voltage
+=============
+This commands the module to a specific drive voltage. A positive voltage always goes in the counterclockwise direction, and a negative voltage always goes clockwise.
+
+ESC Input
+=============
```

Comment:
Also, this assumes a speed module. Servos won't have ESC input. I think you were trying to be style agnostic on this page.

You may want to move this out to a getting started with speed page, which I think you planned on doing at some point in the future but I'm not sure. Can just have things shared by all modules like voltage, velocity, PWM. Or at least specify this is only for speed modules, but then it would also seem like you should talk about the servo input parser testing for servos, so it may get a bit wonky